src/preamble.js

```diff
@@ -936,7 +919,13 @@ function getWasmImports() {
 
 // Create the wasm instance.
 // Receives the wasm imports, returns the exports.
+#if WASM_ASYNC_COMPILATION
+// Funnily enough in JS the `async` keyword has to be on the same line as the
+// function keyword.
+async function createWasm() {
```

Comment:
Alternatively this could be `  {{{ asyncIf(WASM_ASYNC_COMPILATION) }}} function createWasm() {`

Reply:
Nice!  Thats much better